### ISSUE TITLE
accuracy on Pascal VOC

### ISSUE THREAD

**Issue by ds2268** [thomasjpfan/pytorch_refinenet#1]
Hi!

Have you tested this implementation on the Pascal VOC dataset (or any other) and did you compare the results with the original code? (IoU scores would be helpful)

This information would be really helpful on the first page :)

**Comment by thomasjpfan** [comment]
This repo only implements the model. I left it up to the user to perform training with it. https://github.com/dnouri/skorch or https://github.com/inferno-pytorch/inferno are good frameworks to use for training pytorch models.

**Comment by liuchuanloong** [comment]
have you train the loss convergence， my code did not work
